app/uk/gov/hmrc/lisaapi/controllers/JsonFormats.scala

```diff
@@ -189,16 +189,23 @@ trait JsonFormats {
     (JsPath \ "bonuses").read[Bonuses]
   )(RequestBonusPaymentRequest.apply _)
 
+
   implicit val requestBonusPaymentWrites: Writes[RequestBonusPaymentRequest] = (
-    (JsPath \ "lifeEventId").writeNullable[String] and
-    (JsPath \ "periodStartDate").write[String].contramap[DateTime](d => d.toString("yyyy-MM-dd")) and
-    (JsPath \ "periodEndDate").write[String].contramap[DateTime](d => d.toString("yyyy-MM-dd")) and
-    (JsPath \ "transactionType").write[String] and
-    (JsPath \ "htbTransfer").writeNullable[HelpToBuyTransfer] and
-    (JsPath \ "inboundPayments").write[InboundPayments] and
-    (JsPath \ "bonuses").write[Bonuses]
+  (JsPath \ "lifeEventId").writeNullable[String] and
+  (JsPath \ "periodStartDate").write[String].contramap[DateTime](d => d.toString("yyyy-MM-dd")) and
+  (JsPath \ "periodEndDate").write[String].contramap[DateTime](d => d.toString("yyyy-MM-dd")) and
+  (JsPath \ "transactionType").write[String] and
+  (JsPath \ "htbTransfer").writeNullable[HelpToBuyTransfer] and
+  (JsPath \ "inboundPayments").write[InboundPayments] and
+  (JsPath \ "bonuses").write[Bonuses]
   ){req: RequestBonusPaymentRequest => (req.lifeEventId, req.periodStartDate, req.periodEndDate, "Bonus", req.htbTransfer, req.inboundPayments, req.bonuses)}
 
```

Comment:
I can take it out but it was there to test a returned error object when the tests were in the InvestorController